src/pages/experiments/[experimentId]/plots-and-tables/embedding-continuous/index.jsx

```diff
@@ -27,40 +24,38 @@ import { plotNames } from 'utils/constants';
 
 const { Panel } = Collapse;
 
-// TODO: when we want to enable users to create their custom plots,
-// we will need to change this to proper Uuid
 const plotUuid = 'embeddingContinuousMain';
 const plotType = 'embeddingContinuous';
+const PROPERTIES = ['dispersions'];
+const tableState = {
+  pagination: {
+    current: 1, pageSize: 1, showSizeChanger: true, total: 0,
+  },
+  geneNamesFilter: null,
+  sorter: { field: PROPERTIES[0], columnKey: PROPERTIES[0], order: 'descend' },
+};
 
 const ContinuousEmbeddingPage = ({ experimentId }) => {
   const dispatch = useDispatch();
   const config = useSelector((state) => state.componentConfig[plotUuid]?.config);
   const loadedGene = useSelector((state) => state.genes.expression.views[plotUuid]?.data);
   const cellSets = useSelector(getCellSets());
-  useEffect(() => {
-    dispatch(loadPlotConfig(experimentId, plotUuid, plotType));
-    dispatch(loadCellSets(experimentId));
-  }, []);
 
   const geneExpression = useSelector((state) => state.genes.expression);
   const fetching = useSelector((state) => state.genes.properties.views[plotUuid]?.fetching);
   const highestDispersionGene = useSelector(
     (state) => state.genes.properties.views[plotUuid]?.data[0],
   );
 
-  const PROPERTIES = ['dispersions'];
-  const tableState = {
-    pagination: {
-      current: 1, pageSize: 1, showSizeChanger: true, total: 0,
-    },
-    geneNamesFilter: null,
-    sorter: { field: PROPERTIES[0], columnKey: PROPERTIES[0], order: 'descend' },
-  };
-
   const [searchedGene, setSearchedGene] = useState();
 
   useEffect(() => {
-    if (!_.isNil(config?.shownGene) && !searchedGene) {
+    dispatch(loadPlotConfig(experimentId, plotUuid, plotType));
```

Comment:
I think you should load this only if redux is not empty, no? Otherwise we do redundant calls

Reply:
hmm... yea. I'll put guard clauses around these `loadPlotConfig` calls.